cobra2snooty.go

```diff
@@ -119,9 +122,40 @@ func GenDocs(cmd *cobra.Command, w io.Writer) error {
 	}
 	printOptions(buf, cmd)
 
-	if len(cmd.Example) > 0 {
+	if len(s) > 0 && len(s) < 2 {
+		if len(cmd.Example) > 0 {
+			buf.WriteString(examplesHeader)
+			buf.WriteString(fmt.Sprintf("\n%s\n", indentString(cmd.Example, " ")))
+		}
+	}
+
+	if len(s) > 1 {
```

Comment:
I have a feeling all the length checks can be a loop 
```suggestion
	for _, example := range examamples {
```
(the first `_` is the index but I don't think you need it so `_` is the way to ignore it)

Reply:
this would also remove your magic number problems